bitnami/mariadb-galera/README.md

```diff
@@ -349,6 +352,41 @@ extraContainers:
       memory: 10Mi
 ```
 
+### Forcing Bootstraping
+
+> Some of this procedures can lead to data loss, always make a backup beforehand.
+
+To restart the cluster you need to check state in which it is after being stopped, also you will need the previous password for the `rootUser` and `mariabackup`, and the deployment name. 
+You need to check the value of `safe_to_bootstrap` in `/bitnami/mariadb/data/grastate.dat`.
```

Comment:
You need to add more context here...

You're talking about staring the cluster after being stopped but... Why was it stopped? Did I stopped it? Was there an accident? Why do I need to check `safe_to_bootstrap` value?

Reply:
It doesn't matter, this can happen always, either because an nice stop or because a failure.

You need to check the value to find the node to start from. I added some more info regarding this.